py/symbolic/unreal.py

```diff
@@ -0,0 +1,21 @@
+from symbolic._lowlevel import lib, ffi
+from symbolic.utils import RustObject, rustcall
+
+__all__ = ['Unreal4Crash']
+
+class Unreal4Crash(RustObject):
```

Comment:
This needs a

```py
    __dealloc_func__ = lib.semaphore_unreal4crash_free
```

Reply:
good catch. Except u've been doing too much `semaphore` hehe
`symbolic_unreal4_crash_free`